overlay/mce-operator-bundle/mce-operator-bundle-2.7/releaseplan-dev.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+metadata:
+ labels:
+   release.appstudio.openshift.io/auto-release: 'true'
+   release.appstudio.openshift.io/standing-attribution: 'true'
+   release.appstudio.openshift.io/releasePlanAdmission: 'rpa-dev-bundle-mce-27'
+ name: dev-publish-bundle-mce-27
```

Kustomize would append `-mce-27` and make the name `dev-publish-bundle-mce-27-mce-27`
```suggestion
 name: dev-publish-bundle
```